src/api/firebase.js

```diff
@@ -64,6 +62,7 @@ export async function addItem(listId, { itemName, daysUntilNextPurchase }) {
 		name: itemName,
 		totalPurchases: 0,
 	});
+	return data;
```

Comment:
Data in this case is a promise. How can you tell whether it was fulfilled or rejected?